src/octree/raytracing/bevy/types.rs

```diff
@@ -82,6 +85,7 @@ pub struct ShocoVoxRenderData {
     /// | Byte 3  | unused                                                    |
     /// `=====================================================================`
     /// *(1) Only first bit is used in case leaf is uniform
```

Comment:
Am I blind and just not seeing this footnote being referenced anywhere? oO

Reply:
it's not in the changes :)